.github/workflows/publish-components.yaml

```diff
@@ -128,7 +128,7 @@ jobs:
           registry-url: https://registry.npmjs.org/
           cache: 'pnpm'
       - name: Install Dependencies
-        run: npm ci
+        run: pnpm install
```

Comment:
The step below already runs `pnpm install -r` (to install all deps in all workspaces), since we used both `npm` and `pnpm` before. So you can remove this `pnpm install`.